.github/workflows/ci.yml

```diff
@@ -67,7 +72,14 @@ jobs:
             echo $1=$2 >> $GITHUB_ENV
           }
 
-          set_env_var PATH /usr/lib/ccache:$PATH
+          case `uname` in
+              Darwin) ccache_path=/usr/local/opt/ccache/libexec ;;
```

Comment:
Assuming this path is for Homebrew (looks that way), can you add `/opt/homebrew/opt/ccache/libexec` too? That's the location used by Homebrew on ARM Macs.

Reply:
Sorry, momentarily forgot this is for CI (which is unlikely to have ARM anytime soon), so please ignore that.

Reply:
I hadn't even realized that the path was different for ARM Macs (in spite of using one myself...), but I agree that it's not that important for now, as they're not used for the CI, so I've created https://github.com/hendrikmuhs/ccache-action/issues/101 to maybe improve this in the future, thanks for pointing this out!